README.md

```diff
@@ -166,6 +175,10 @@ Parameter | Description | Default
 It is recommended to configure the following variables. It can be done in the
 `values-overrides.yaml` file.
 
+!!! info
```

Comment:
I don't think those kind of containers get rendered in markdown, so I would change it to plain markdown so the file remains readable on github
```suggestion

```

Reply:
I didn't realise there's a difference between the docs and the markdown on github. I removed the line and replaced the blanks by '>' in the next line.